### ISSUE TITLE
[QUESTION]

### ISSUE THREAD

**Issue by mekonnen070** [epam-cross-platform-lab/swagger-dart-code-generator#530]
Hey, Good project. I am having an issue generating the parameters to be in snake_case because my backend keys are written in snake case. it generates the parameters in only camelCase so is there any way I can pass any option to make it so? please help me out with this one.

**Comment by Vovanella95** [comment]
Hi, it's does not matter how parameters generated, because they will have original `JsonKey()`. 

From the model perspective, you will have JsonKey and when toJson will called - generated json will have `group_id: 'someId'`

```dart
  @JsonKey(name: 'group_id', includeIfNull: false, defaultValue: '')
  final String groupId;
```

From the requests perspective, we have `@Header('group_id')`, `@Path('group_id')` and ` Query('group_id')`, and these attributes will pass snake_case value for Chopper.

 ```dart
  @Get(path: '/tasks')
  Future<chopper.Response<TasksGet$Response>> _tasksGet(
      {@Header('group_id') String? group_id});
}
 ```
If I understood your problem wrong - could you provide more details?

**Comment by Vovanella95** [comment]
Hi again @mekonnen070 , this issue is duplicate of #532 
Closing this one. Will be resolved in #532
